### ISSUE TITLE
Inverse tafqit

### ISSUE THREAD

**Issue by OmarHory** [MohsenAlyafei/tafqit#3]
Is it possible to do Inverse-tafqit?
Like from words to numbers.
If yes, can you provide me with an example?

**Comment by MohsenAlyafei** [comment]
> Is it possible to do Inverse-tafqit? Like from words to numbers. If yes, can you provide me with an example?

Yes, it is possible in Arabic and English and I had a plan to get it done.

**Comment by MohsenAlyafei** [comment]
> Is it possible to do Inverse-tafqit? Like from words to numbers. If yes, can you provide me with an example?

Please see this Twitter post:

https://twitter.com/maalyafei/status/1473360246055841796
